index.html

```diff
@@ -35,5 +35,127 @@ <h1>I’m Benson.<br>Glad to see you!</h1>
+                <div class="image1">
+                    <img src="images/Snapshoot Portfolio (4).png" alt="project snapshot4">
+                </div>
+                <div class="body1">
+                    <h2>Multi-Post Stories</h2>
+                    <ul class="flexy">
+                        <li class="canopy">CANOPY</li>
+                        <li class="backend">Back End Dev</li>
+                        <li class="year">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="linkss">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javasript</li>
+                    </ul>
+                    <button>See Project</button>
+                </div>
+            </article>
+        </section>
+        <section class="about">
+            <div class="main">
+                <h1>About Myself</h1>
+                <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            
+                <p class="caps">LET’S CONNECT</p>
+                <ul class="icons">
+                    <li><a href=""><img src="images/Twitter icon.png" alt="twitter"></a></li>
+                    <li><a href="www.linkedin.com/in/bensonkiai"><img src="images/Linkedin icon.png" alt="linkedin"></a></li>
+                    <li><a href=""><img src="images/Email icon.png" alt="Email"></a></li>
+                    <li><a href="https://github.com/BenMKT/Portfolio"><img src="images/Github icon.png" alt="github"></a></li>
+                    <li><a href=""><img src="images/Peace icon.png" alt="site"></a></li>
+                </ul>
+                <button>Get my resume</button>
+                <ul class="last">
+                    <li class="list">Languages<i class="fa">&#xf078</i>
+                        <ul class="drop">
+                            <div class="stretch">
+                                <li><img src="images/JS Ellipse.png" alt="Javascript"></li>
+                            </div>
+                            <div class="stretch">
+                                <li><img src="images/HTML Ellipse.png" alt="HTML"></li>
+                            </div>
+                            <div class="stretch">
+                                <li><img src="images/CSS Ellipse.png" alt="CSS"></li>
+                            </div>
+                        </ul>
+                    </li>
+                    <li class="list">Frameworks<i class="fa">&#xf054</i></li>
+                    <hr class="rule">
+                    <li class="list">Skills<i class="fa">&#xf054</i></li>
+                    <hr class="rule">
+                </ul>
```

Comment:
The list of technologies should be listed in a `<ul> <li> </li></ul> ` tag and not you having different ul's and li's yag in between .Kindly implement this change so you have a clean code that also follows semantics purposes 😄

Reply:
- Removed nested `ul` tags with their accompanying classes and styling for cleaner code in About section.